wagtail/contrib/styleguide/templates/wagtailstyleguide/base.html

```diff
@@ -7,578 +7,658 @@
-                <p>It could be multiple lines</p>
-            {% endhelp_block %}
-
-            {% help_block status="warning" %}
-                A warning message might be output in cases where a user's action could have serious <a href="#help">consequences</a>.
-            {% endhelp_block %}
-
-            {% help_block status="critical" %}
-                A critical message would probably be rare, in cases where a particularly brittle or <a href="#help">dangerously destructive action</a> could be performed and needs to be warned about.
-            {% endhelp_block %}
-
-        {% endpanel %}
-
-        {% panel id="listings" heading="Listings" %}
-
-            <h3><code>table</code> listing</h3>
-            <table class="listing">
-                <thead>
-                    <tr>
-                        <th>Heading 1</th>
-                        <th>Heading 2</th>
-                        <th>Heading 3</th>
-                    </tr>
-                </thead>
-                <tbody>
-                    <tr>
-                        <td class="title">
-                            <h2><a href="">TD with title class</a></h2>
-                        </td>
-                        <td>Standard TD</td>
-                        <td>Standard TD</td>
-                    </tr>
-                    <tr class="unpublished">
-                        <td class="title">
-                            <h2><a href="">Unpublished TD with title class</a></h2>
-                        </td>
-                        <td>Standard TD</td>
-                        <td>Standard TD</td>
-                    </tr>
-                    <tr>
-                        <td class="title">
-                            <h2><a href="">TD with title class</a></h2>
-                        </td>
-                        <td>Standard TD</td>
-                        <td>Standard TD</td>
-                    </tr>
-                </tbody>
-            </table>
-
-            <h3><code>ul</code> listing</h3>
-            <ul class="listing">
-                <li><div class="title"><h2><a href="">List item</a></h2></div></li>
-                <li><div class="title"><h2><a href="">List item</a></h2></div></li>
-                <li><div class="title"><h2><a href="">List item</a></h2></div></li>
-            </ul>
 
+{% block main_content %}
 
-            <h3><code>ul</code> listings with multiple columns</h3>
-            <ul class="listing">
-                <li>
-                    <div class="row row-flush">
-                        <div class="col6 title">
-                            <h2><a href="">Something here</a></h2>
-                        </div>
+    {% trans "Styleguide" as title_trans %}
+    {% include "wagtailadmin/shared/header.html" with title=title_trans icon='image' %}
```

Comment:
We shouldn't need this shared header anymore. Look at the structure of the generic/base.html template to understand.

Reply:
Oooh. Right!!
I'll do my homework.

Reply:
Homework done! I read through the `generic/base.html` template and saw what you meant.